docker-compose.yaml

```diff
@@ -47,7 +47,8 @@ services:
       <<: *jellyfish-environment
       RELEASE_NODE: app@app2
       NODE_NAME: app@app2
-      PORT: 4002
+      JF_HOST: "localhost:4002"
```

Comment:
I am not sure if I am gonna introduce this change as IMO its misleading now :thinking:

Reply:
If we had 4 vars we could get rid of `:4002` but let's try with 3 vars and see how it works